CHANGELOG.md

```diff
@@ -69,6 +69,8 @@ v0.29.0-rc.0 (2022-11-03)
   - `otelcol.exporter.prometheus` forwards OTLP-formatted data to compatible
     `prometheus` components. (@rfratto)
 
+  - `grafana-agent-operator` supports oauth2 as an authentication method for remote_write. (@timo-42)
```

Comment:
This has to be sync with last changes from master

Reply:
Did a rebase and push

Reply:
I'm sorry for the back and forth, but could you move it to `Main (unreleased)` section? thanks!